packages/scripts/src/create-webpack-configs.ts

```diff
@@ -276,8 +276,10 @@ export function createWebpackConfigForExteranlFeature({
             path: outputPath,
             filename: `[name].${target}.js`,
             chunkFilename: `[name].${target}.js`,
-            libraryTarget: 'var',
-            jsonpFunction: packageName + name,
+            library: {
+                type: 'global',
```

Comment:
why did you change to 'global'?

Reply:
got a runtime error when using type: `jsonp` with that name. maybe I'm missing something obvious.

Reply:
from what I read at https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-unique-naming I see that the `jsonpFunction` is no longer needed.
maybe just restore the `libraryTarget: 'var'` (https://webpack.js.org/configuration/output/#expose-a-variable) and remove the `jsonpFunction`

Reply:
will try

Reply:
@AviVahl seems like removing the libraryTarget whatsoever fixed the issue, and now tests pass